### PR TITLE
Add missing arg value in multi-app-template.md

### PR DESCRIPTION
**Pull request by stuartleeks**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added a missing arg placeholder for `dapr stop -f` example

### CODE REVIEW

**Review by hhunter-ms — APPROVED** [review]

**Comment by stuartleeks** [comment]
I see that the `validate-links` check has failed, but I haven't made any links edits in this PR 😄

**Comment by hhunter-ms** [comment]
@stuartleeks hm I'm not sure why the link isn't working - it works for me. try bookending the 'broken' links with these:

> < !-- IGNORE_LINKS -- >
> < !-- END_IGNORE -- >

**Comment by stuartleeks** [comment]
@hhunter-ms - it turns out that the broken link has a 301 redirect, so I've updated the links to the new URL

File: daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This component supports both **input and output** binding interfaces.
 {{< tabs "Self-Hosted" "Kubernetes">}}
 
 {{% codetab %}}
-1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/login/register](https://account.kubemq.io/login/register) and register for a key.
+1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md:55]:
```suggestion
<!-- IGNORE_LINKS -->
1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
<!-- END_IGNORE -->
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ You can then interact with the server using the client port: `localhost:50000`
 {{% /codetab %}}
 
 {{% codetab %}}
-1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/login/register](https://account.kubemq.io/login/register) and register for a key.
+1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md:70]:
```suggestion
<!-- IGNORE_LINKS -->
1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
<!-- END_IGNORE -->
```

**Pull request merged**